modules/basic-skills/src/views/full/callApi.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import { Row, Col, Label, Input } from 'reactstrap'
+import Select from 'react-select'
+import style from './style.scss'
+import { BotpressTooltip } from 'botpress/tooltip'
+import { LinkDocumentationProvider } from 'botpress/documentation'
+
+const methodOptions = [
+  { label: 'get', value: 'get' },
```

Comment:
I know that's picky but can you set the labels to uppercases :)

Reply:
Solved!